src/app/components/Embeds/EmbedImages/index.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { PropsWithChildren, useContext } from 'react';
+// import { OptimoBlock } from '#app/models/types/optimo';
+import { RequestContext } from '#app/contexts/RequestContext';
+import Image from '../../Image';
+// import styles from './index.styles';
```

Comment:
Should we remove these commented out lines?

Reply:
Ahh, accidentally did the commits in the AMP PR, all the comments will get addressed in there https://github.com/bbc/simorgh/pull/10965